src/main.ts

```diff
@@ -185,6 +186,7 @@ async function getTsServerRepoResult(
     newTsServerPath: string,
     downloadDir: string,
     replayScriptArtifactPath: string,
+    rawErrorScriptArtifactPath: string,
```

Comment:
Sorry if my previous request was unclear, but can you please change "rawErrorScript" to "rawError"?

Reply:
Done :)